client/gutenberg/editor/calypsoify-iframe.tsx

```diff
@@ -346,8 +346,8 @@ class CalypsoifyIframe extends Component<
 					post: translate( 'View all posts' ),
 				},
 				createPostLabels: {
-					page: translate( 'Create new page' ),
-					post: translate( 'Create new post' ),
+					page: translate( 'Add new page' ),
+					post: translate( 'Add new post' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **28** times:
`translate( 'Add a New Post' )` *ES Score: 6*